docs/index.md

```diff
@@ -3,12 +3,146 @@
-- create new workflows that use git repositories
-- re-run jobs you've created and jobs others have created
-- download and restore workflows you've created and workflows you've run
-
-Downloaded storage locations and worflows are in JSON format and can be shared with others.
+The UI provides a convienient way for running workflows.
+It also provide ways to:
+
+ * configure one or more storage locations at the folder and file levels
+ * download and restore configured storage locations
+ * run workflows
+ * create new workflows that use git repositories
+ * re-run jobs you've created and jobs others have created
+ * download and restore workflows you've created and workflows you've run
+
+The downloaded configurations of storage locations and workflows are in JSON format and can be shared with others.
+
+## Tutorial
+Items needed:
+
+- an orthomosaic image to be processed
+- a GeoJSON file containing the plots of interest
+- an optional YAML file containing [experiment information](https://osf.io/xdkcy/wiki/Configuration%20YAML/) [![external link](img/external_link.png)](https://osf.io/xdkcy/wiki/Configuration%20YAML/)
+
+What we will  be doing:
+
+1. [Upload files and configure](#uploading-and-configuring-files) a server-side file storage definition
+2. Running the [Canopy Cover image-based workflow](#running-a-workflow)
+3. [Downloading the results](#downloading-the-results) to the local computer
+4. [Deleting](#delete-workflow-run) the workflow run
+
+Note that the geographic boundaries of the image don't need to match that of the plots.
+Only those plots that intersect the image will be processed.
+
+Additionally, the coordinate system of the image doesn't need to match that of the plots.
+Plot points are converted automatically to the image's geographic coordinate system.
+
+### Sample data
+
+We also provide an [archive of sample data](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+) [![external link](img/external_link.png)](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz) that can be downloaded and used in this Tutorial.
+Be sure to extract the files from this archive before uploading them.
+Uploaded archives are **not** usable by the system.
+
+The following `bash` commands can be used to download the archive and extract the files.
+
+```bash
+curl --output sample_website_data.tar.gz https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+tar -xzf sample_website_data.tar.gz
+```
+
+### Uploading and configuring files
+
+On the main page of the site, click the `files` link under **Data Sources**.
+This will take you to the file storage definition page
+
+<img src="img/landing_click_files.png" width="450" alt="File storage definition link" aria-label="File storage definition link" title="Click on Files link"/>
+
+Next click the upload icon, select and upload the files to process.
```

Comment:
the upload icon is too faint; I didn't notice it at first

Reply:
Issue: https://github.com/AgPipeline/issues-and-projects/issues/560